modules/ROOT/pages/platform/security/single-sign-on.adoc

```diff
@@ -5,49 +5,57 @@
 label:AuraDB-Enterprise[]
 label:AuraDS-Enterprise[]
 
-Aura Enterprise supports Single Sign-On (SSO) at both the Console level and for accessing Workspace, Bloom and Browser clients directly at the instance level.
+== SSO levels
+
+Aura admins can configure organization and tenant level SSO:
+
+* *Organization level SSO:* Applies to everything related to the console within an organization. Grants console access for organization management, this includes editing organization and tenant settings, viewing the list of instances within tenants, and managing instances (such as pause/resume). However, it does not grant access to view or modify data within the database instances themselves. Currently, role mapping is not applicable to Organization level SSO.
```

Comment:
> Grants console access for organization management, this includes editing organization and tenant settings, viewing the list of instances within tenants, and managing instances (such as pause/resume).

So this isn't really correct because just because you can login via SSO doesn't mean you have access to the org. You still have to be invited and the invitee will grant you a role when you are invited and that dictates if you can edit settings or view/manage instances. Basically org sso as it is today simple allows org admins to restrict how users login when they are trying to access the org. Everything else around access is managed via roles/invites.